blocks/rich-text/index.js

```diff
@@ -212,6 +219,23 @@ export default class RichText extends Component {
 	}
 
 	onFocus() {
+		// For virtual keyboards, always scroll the focussed editor into view.
+		// Unfortunately we cannot detect virtual keyboards, so we check UA.
```

Comment:
I don't have an external keyboard right now, but what happens if I use Gutenberg with a mobile device with an external keyboard? Most systems hide the virtual keyboard whenever an external keyboard is connected; any chance the auto-scrolling can be detrimental in those cases?

Reply:
It wouldn't be _super_ annoying I think. The focussed field won't go out of view, and in these cases, we're still dealing with a relatively small screen. The bigger the screen, the worse this gets. While not ideal, this is the best I can come up with. I've played with other approaches as well, e.g. trying to detect touch (vs clicks), but failed to make it work. The order of which touch, click, and focus events are fired makes it seems impossible to do.